torch/_inductor/codegen/mps.py

```diff
@@ -183,6 +183,10 @@ def sqrt(x: CSEVariable) -> str:
     def rsqrt(x: CSEVariable) -> str:
         return f"metal::rsqrt({x})"
 
+    @staticmethod
+    def tanh(x: CSEVariable) -> str:
+        return f"metal::tanh({x})"
```

Comment:
I'm curious why `metal::precise::tanh` is not needed here

Reply:
Maybe there's some test that might fail due to the lack of accuracy, but `tanh` pass. Is there a way we can verify this hypothesis? We might write something that generates random vector and compares until there's a failure.